experiments/modern/src/App.js

```diff
@@ -34,19 +37,16 @@ const IGNORE_IDS = new Set([
 const SkinContext = React.createContext(null);
 
 async function getSkin() {
-  const resp = await fetch(process.env.PUBLIC_URL + "/skins/CornerAmp_Redux.wal");
+  // const resp = await fetch(process.env.PUBLIC_URL + "/skins/CornerAmp_Redux.wal");
+  const resp = await fetch(process.env.PUBLIC_URL + "/skins/simple.wal");
```

Comment:
We should also add support for dragging a skin in. (In a subsequent PR)

Reply:
Yes! that would be convenient and also help make sure we have things properly cleaning up!